modules/cactus-web/src/BrandBar/BrandBar.tsx

```diff
@@ -337,6 +337,13 @@ const ActionMenuList = styled.ul`
   padding: 0;
   margin: 0;
   outline: none;
+  word-break: break-word;
```

Comment:
Remember that this doesn't work in IE11.  There are a couple of other places in this file that should be corrected as well.

```suggestion
  word-wrap: break-word;
  overflow-wrap: break-word;
```